crypto/evp/p_open.c

```diff
@@ -8,15 +8,12 @@
  */
 
 #include "internal/cryptlib.h"
-#ifdef OPENSSL_NO_RSA
```

Comment:
Not reflected in evp/build.info changes.

Reply:
rebase glitch; fixed.